lib/capistrano/db_sync/postgres/cli.rb

```diff
@@ -64,7 +64,9 @@ def credentials
   end
 
   def with_pw
-    "PGPASSWORD='#{config['password']}'" unless config.fetch('password', '').empty?
+    unless (config['password'] || '').empty?
```

Comment:
I love and prefer using `Hash#fetch` over `Hash#[]` since it enforces a value with a possible default, but with a `nil` value, the default value is not used and the `empty?` method is raising the error.

That's why I've chose to use the `Hash#[]` variant to have the original value untouched, of a nil if missing. Then the `||` sets the default value.

The whole `String#empty?` portion could also be removed, sinc I guess we can rely on the falsiness of a nil or missing value.